docs/src/paradox/03-apis/api-v2/tei-xml.md

```diff
@@ -0,0 +1,433 @@
+      </titleStmt>
+      <publicationStmt>
+        <p>
+             This is the TEI/XML representation of a resource identified by the Iri http://rdfh.ch/0001/thing_with_richtext_with_markup.
+         </p>
+      </publicationStmt>
+      <sourceDesc>
+        <p>Representation of the resource's text as TEI/XML</p>
+      </sourceDesc>
+    </fileDesc>
+  </teiHeader>
+  <text>
+    <body>
+      <p>This is a test that contains marked up elements. This is <hi rend="italic">interesting text</hi> in italics. This is <hi rend="italic">boring text</hi> in italics.</p>
+    </body>
+  </text>
+</TEI>        
+```
+
+The body of the TEI document contains the standoff markup as XML. The header contains contains some basic metadata about the resource such as the `rdfs:label` an its IRI. However, this might not be sufficient for more advanced use cases like digital edition projects. 
+In that case, a custom conversion has to be performed (see below).
+
+## Custom Conversion
+
+If a project defines its own standoff entities, a custom conversion can be provided (body of the TEI document). Also for the TEI header, a custom conversion can be provided.
+
+For the custom conversion, additional configuration is required.
+
+TEI body:
+    
+- additional mapping from standoff to XML (URL parameter `mappingIri`)
+- XSL transformation to turn the XML into a valid TEI body (referred to by the mapping).
+
+The mapping has to refer to a `defaultXSLTransformation` that transforms the XML that was created from standoff markup (see @ref:[XML To Standoff Mapping in API v1](../api-v1/xml-to-standoff-mapping.md)). This step is necessary because the mapping assumes a one to one relation between standoff classes and properties and XML elements and attributes.
+For example, we may want to convert a `standoff:StandoffItalicTag` into TEI/XML. TEI expresses this as `<hi rend="italic">...</hi>`. In the mapping, the `standoff:StandoffItalicTag` may be mapped to a a temporary XML element that is going to be converted to `<hi rend="italic">...</hi>` in a further step by the XSLT. 
+
+For sample data, see `webapi/_test_data/test_route/texts/beol/BEOLTEIMapping.xml` (mapping) and `webapi/_test_data/test_route/texts/beol/standoffToTEI.xsl`. The standoff entities are defined in `beol-onto.ttl`.
+
+TEI header:
+    
+- Gravsearch template to query the resources metadata, results are serialized to RDF/XML (URL parameter `gravsearchTemplateIri`)
+- XSL transformation to turn that RDF/XML into a valid TEI header (URL parameter `teiHeaderXSLTIri`)
+
+The Gravserarch template is expected to be of type `knora-base:TextRepresentation` and to contain a placeholder `$resourceIri` that is to be replaced by the actual resource Iri.
+The Gravsearch template is expected to contain a query involving the text property (URL parameter `textProperty`) and more properties that are going to be mapped to the TEI header. The Gravsearch template is a simple text file with the files extension `.txt`.
```

Comment:
`.rq` might be better than `.txt`:

https://www.w3.org/TR/sparql11-query/#mediaType

> It is recommended that sparql query files have the extension ".rq" (lowercase) on all platforms.

Reply:
Since it is a template I thought it makes more sense to use ".txt". The placeholder has to be replaced before it is valid SPARQL.

We could call it `gravsearch.rq.txt` in analogy to our twirl templates in scala.

Reply:
Let's make a Gravsearch MIME type, but not right now. :)

Reply:
see https://github.com/dhlab-basel/Sipi/issues/238